### ISSUE TITLE
ERA5 preprocessing: consistent time coordinates

### ISSUE THREAD

**Issue by BSchilperoort** [WAM2layers/WAM2layers#360]
The ERA5 data used in the preprocessor has some quirks.

Accumulated variables are from the preceding hour (e.g. precipitation, evaporation).
Instantaneous variables (moisture content, wind) are "at the timestamp".

This causes a misalignment if you want to define time stamps. The rest of the code in WAM2layers also assumes that all input variables have the same time bounds.

To solve this issue:

- [ ] Homogenize time coordinates for all ERA5 variables
- [ ] Add these time bounds to the output data

**Comment by ruudvdent** [comment]
I agree, this is important, not only for the output of the pre-processing data itself, but in fact it is also something ambigous what *_end_date in the config means precisely.

**Comment by ruudvdent** [comment]
We agree that it would be nice to make ERA5 follow CMIP via ESMVALTOOL

**Comment by ruudvdent** [comment]
Important to check how this works through in tracking. Could potentially destroy backward compatibility.
